releases/tex/master/extra/crowbar-backup

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+FILE="/root/crowbar_backup.tar.gz"
+if [ -f "$FILE" ]; then
+    echo "$FILE already exists."
+    read -p "Do you want to overwrite $FILE? (y/N) "
+    if [[ $REPLY != [yY] ]] ; then
+    exit 0
+    fi
+fi
+
+tmpdir=$(mktemp -d)
```

Comment:
You should probably use `trap` to cleanup and remove that directory in case the script is interrupted.